### ISSUE TITLE
Fill in the blank questions not working

### ISSUE THREAD

**Issue by Geometryke** [smintf/autolingo#25]
The extension seems to halt function once it reaches a question that asks to fill in the blank.

**Comment by nikitos123456789** [comment]
Yeah I’m having the same issue as it answer a couple normal questions at the beginning of the questions and then is not able to answer the fill in the black questions.
Is this similar to what you’re having?
